test/resque_failure_multiple_test.rb

```diff
@@ -62,3 +64,22 @@ def save
     end
   end
 end
+
+describe "blacklisted_exceptions" do
+  it "sets blacklisted_exceptions on the multiple backend" do
+    blacklisted_exceptions = [CustomException].map(&:name).map(&:to_s)
```

Comment:
I think that we don't need the `to_s` call here since name already returns a string, right?

Reply:
Yes `to_s` is not necessary but it is nice to have to ensure we get a string! I can remove it as this is in a test file!

Reply:
I don't really agree that it's nice to have. Since the arguments are already converted to string we don't have to perform extra iterations on the list.

Reply:
Another note here `map` creates an extra (useless in this case) array